java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/LiveDeployment/LogFilterDialog.java

```diff
@@ -97,10 +97,10 @@ public void actionPerformed(ActionEvent e) {
                             return;
                         }
 
```

Comment:
Most of the curly-brace violations were because of how we put comments at the end of a statement.
All of these had to be fixed by hand.